etcd.tf

```diff
@@ -14,6 +14,7 @@ data "template_file" "etcd-cfssl-new-cert" {
     path    = "/etc/etcd/ssl"
     cn      = "${count.index}.etcd.${var.dns_domain}"
     org     = ""
+    get_ip  = "${var.get_ip_command[var.cloud_provider]}"
```

Comment:
Why don't you pass in the provider here and have the logic of which command to use in the template, didn't want to replicate the logic?